### DISCUSSION TITLE
注册用户登录后，首页文章不显示

### DISCUSSION THREAD

**Discussion by eazhijia** [Tokinx/Wing#19] [讨论]
你好，我用的你的主题，首页也是做成了你的那种，就是首页显示的是“笔记的页面”，网站开启了注册功能，但是当用户注册登录后，首页的文章就不都显示了，这个怎么开启无论登录与否都显示全部文章和所有笔记呢

**Comment by Tokinx** [comment]
这个你提一下issue吧，方便的话可以录个video嘛？我似懂非懂。
只是近期电脑出掉了，所以这个问题可能要下个月才会安排时间来修复

**Reply by ssxzi** [reply]
#144
